From d79cc91e6fd778ebc069d53fefd8495dbf7549fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Javier=20Jard=C3=B3n?= <jjardon@gnome.org>
Date: Sun, 2 Oct 2011 23:31:59 +0100
Subject: [PATCH] configure.ac: Use a auxiliar directory to store
 autogenerated files (cherry picked from commit
 a80db8bd5f7f15859e8891aab9fc3694ce4cd0bd)

---
 configure.ac |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index bb01b46..bee45da 100644
--- a/configure.ac
+++ b/configure.ac
@@ -26,6 +26,8 @@ AC_INIT([systemd],
 AC_CONFIG_SRCDIR([src/core/main.c])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_AUX_DIR([build-aux])
+
 AC_USE_SYSTEM_EXTENSIONS
 AC_SYS_LARGEFILE
 AC_PREFIX_DEFAULT([/usr])
